server/testing/src/main/java/org/apache/james/util/docker/RateLimiters.java

```diff
@@ -29,4 +29,9 @@
         .withRate(20, TimeUnit.SECONDS)
```

Comment:
_DEFAULT_ is not a good name, IMO.
Client of this constant doesn't know it's real value.